lib/src/git_backend.rs

```diff
@@ -838,7 +838,6 @@ impl Backend for GitBackend {
         };
         // Update the signatures to match the ones that were actually written to the
```

Comment:
nit: i suppose this should be singular "the signature to match the one that was" now

Reply:
fixed